test/test_file_digest.py

```diff
@@ -221,6 +221,119 @@ def test_multiple_packages():
                 assert error.startswith(f'{pkgname}: E: somerestriction-file-unauthorized /restricted/1/afile')
 
 
+SHELL_SCRIPT = '''#!/bin/bash
+# let's iterate over 1 to 10
+for i in `seq 10`; do
+    echo "$i"
+done
+
+fun() {
+    # what fun?!
+    echo "have fun!"  
+}'''  # noqa: W291
+
```

Comment:
Please move this to separate files so that you don't have to use `noqa`. Similarly for expected output.